core/scripts/ocr2vrf/util.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+
+	helpers "github.com/smartcontractkit/chainlink/core/scripts/common"
+	"github.com/smartcontractkit/ocr2vrf/gethwrappers/dkg"
+	"github.com/smartcontractkit/ocr2vrf/gethwrappers/vrf"
+)
+
+func DeployDKG(e helpers.Environment) (blockhashStoreAddress common.Address) {
+	_, tx, _, err := dkg.DeployDKG(e.Owner, e.Ec)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func AddClientToDKG(e helpers.Environment, dkgAddress string, keyId string, clientAddress string) {
```

Comment:
Naming: `keyID` rather than `keyId`. This _should_ be enforced by `gofmt` already, but not sure.